src/xpdf-4.04/aconf.h

```diff
@@ -29,7 +29,7 @@
 /*
  * Enable multithreading support.
  */
-#define MULTITHREADED 0
+#define MULTITHREADED 1
```

Comment:
Let's keep this at 0

Reply:
I agree, but since i want to use it in a multithreaded app i like to set it to enabled